### ISSUE TITLE
Export old data from the last two years

### ISSUE THREAD

**Issue by carolyncole** [pulibrary/locker_and_study_spaces#3]
We need to export data from the old system including the last two years of information.

This will depend on the data model in #2

It should be repeatable as this is an active system that will be in use while we transition.  We should be able to get a dump for testing and then another dump when we go live.

**Comment by carolyncole** [comment]
fixed by 3391fb1b44987ed79a3a1a61f18b302019ed897e

**Comment by carolyncole** [comment]
Updated query to get lat two years and any application that is assigned, but not expired.
```
SELECT Applicant.*, Status.status as statusAtApplication, Location.location as preferredGeneralArea, Department.deptTitle as departmentAtApplication, Semester.semester, Spaces.space, Spaces.location, SpaceType.type FROM [Applicant] 
join Status on Applicant.status = Status.id 
join Location on Applicant.locationPreference = Location.id 
join Department on Applicant.deptID = Department.deptId 
join Semester on Applicant.occupancySemester = Semester.id 
left join Spaces on Applicant.spaceAssigned = Spaces.spaceId
join SpaceType on Applicant.spaceTypePreference = SpaceType.id
where date_of_application > '20190101' or (releaseDate is null and assignedDate is not null and expirationDate > CURRENT_TIMESTAMP)
```
